static/app/views/monitors/monitors.tsx

```diff
@@ -80,9 +77,13 @@ class Monitors extends AsyncView<Props, State> {
     return `Monitors - ${this.orgSlug}`;
   }
 
-  componentDidMount() {
-    this.props.setEventNames('monitors.page_viewed', 'Monitors: Page Viewed');
+  onRequestSuccess(response): void {
+    trackAdvancedAnalyticsEvent('monitors.page_viewed', {
```

Comment:
@scefali don't we have a diff way to do this now

Reply:
@evanpurkhiser yes, see the docs: https://develop.sentry.dev/analytics/#route-based-frontend-analytics

Reply:
ahh I see, even though we wait for a request to resolve first I should still use the automated route analytics right? I guess that is handled by the 7s delay, is that correct?

Reply:
@davidenwang yes, exactly. As long as it happens within 7s then it will just work.

Reply:
okay will change accordingly